### PR TITLE
5146: Logout route

### PR DESCRIPTION
**Pull request by kat1906**

### CODE REVIEW

File: app/includes/header.njk [hunk 1]
```diff
@@ -23,9 +23,9 @@
   dataTestId: "header-banner",
   innerComponent: loginLogout({
     dataTestId: "login-logout-component",
-    username: username,
+    username: 'kk',
```
Comment (alibombaye) [inline app/includes/header.njk]: just one `k` off a warning 👿

Comment (alibombaye) [reply app/includes/header.njk]: actually why has this been changed?

Comment (kat1906) [reply app/includes/header.njk]: so I could see it was working - need to revert

**Comment by kat1906** [comment]
> Looks good. Just your choice if you want to update the username

What do you mean? I just did it to check it was working as it needed to be logged in to show the log out button

**Pull request merged**